.gitignore

```diff
@@ -17,3 +17,4 @@ build/
 
 # Marquez configuration
 config.yml
+node_modules
```

Comment:
_Minor_: We don't need to ignore `node_modules`

Reply:
ah sorry about that

Reply:
No problem at all!